### PR TITLE
Feature/allow to specify trust server

### PR DESCRIPTION
**Pull request by jw-s**
Addresses #22 

FYI, looks like the e2e tests are broken (Fails when following the e2e README). So even though I have tests they're unable to execute. Will look into what is causing the problem.

### CODE REVIEW

**Comment by liamawhite** [comment]
Only just saw this! Code looks good.

The e2e tests can only really be run on an IKS cluster due to image pull secrets. So I will give them a run when I have a chance to in the next few days.

cc @molepigeon

**Comment by molepigeon** [comment]
Yes, I've been tripped up by the e2e tests too. Turns out they need to run in an IKS cluster _in our account_, because the images used in the tests are private.

In future we should make it so that anyone can run the e2e tests without special auth tokens. I raised #34 to address this.

File: pkg/controller/notary/controller.go [hunk 1]
```diff
@@ -155,10 +155,14 @@ func (c *Controller) mutatePodSpec(namespace, specPath string, pod corev1.PodSpe
 
 				// Get image digest
 				glog.Info("getting signed image...")
-				notaryURL, err := img.GetContentTrustURL()
-				if err != nil {
-					a.StringToAdmissionResponse(fmt.Sprintf("Trust Server/Image Configuration Error: %v", err.Error()))
-					continue containerLoop
+
+				notaryURL := policy.Trust.TrustServer
```
[the remark this answers is not in the document]
Comment (jw-s) [reply pkg/controller/notary/controller.go]: Done

**Comment by jw-s** [comment]
Also you'll notice more files have changed. The code wasn't formatted according to `gofmt`.

**Comment by liamawhite** [comment]
LGTM, @molepigeon any objections?

**Comment by liamawhite** [comment]
Could you add a PR to update usage instructions in the README as well? 🙂

**Pull request merged**